rule/flags/flags.go

```diff
@@ -80,6 +85,10 @@ func Parse(s string) (rule.Rule, error) {
 		} else if ruleFlagSet.Type == rule.PrependSyscallRuleType {
 			syscallRule.List = ruleFlagSet.Prepend.List
 			syscallRule.Action = ruleFlagSet.Prepend.Action
+		} else if ruleFlagSet.Type == rule.DeleteSyscallRuleType {
+			syscallRule.List = ruleFlagSet.Delete.List
+			syscallRule.Action = ruleFlagSet.Delete.Action
+			fmt.Printf("Parsed %s syscalls\n", syscallRule.Syscalls)
```

Comment:
I think at this point, this becomes a switch.

Reply:
Done.